c++/src/Reader.hh

```diff
@@ -19,13 +19,14 @@
 #ifndef ORC_READER_IMPL_HH
 #define ORC_READER_IMPL_HH
 
+#include "orc/Exceptions.hh"
 #include "orc/Int128.hh"
 #include "orc/OrcFile.hh"
 #include "orc/Reader.hh"
 
 #include "ColumnReader.hh"
-#include "orc/Exceptions.hh"
```

Comment:
The move from line 27 to 22 looks reasonable. BTW, do we have a C++ convention for `C++ include` statement order?

Reply:
Unfortunately I don't think we had a clear C++ coding style. Maybe we can use cpplint tool to enfoce it.

Reply:
Thanks. No problem then. I'll ignore the C++ coding style.